### DISCUSSION TITLE
Redefine Quorum for DCDAO Voting

### DISCUSSION THREAD

**Discussion by 2xJDubs** [ringdao-community-guild/discussion#52] [Ideas]
The current DCDAO vote regarding the Steering Committee only required a Quorum of 4 members.  This number made sense before new members outside of the core team were added to DCDAO.  Now that we have 11 members of DCDAO we should consider changing quorum to hold with the typical definition of 2/3rds majority.  Below is a definition from wikipedia to show the percentage usually used for quorum.

![image](https://github.com/dcdao/discussion/assets/35436605/2a7770fe-621f-4c88-984e-b789754aab85)

Making this change makes sure that members outside of the core team have a voice in votes and helps to engage the community.  Along with this change we should consider votes that are not time sensitive have a 7 day voting period.

**Comment by SasoLithops** [comment]
I absolutely agree with what Jesse proposes. To raise quorum to 2/3 of members and increase voting time to 7 days as few days is to short.
We could take snapshot of number of members before vote goes live and 2/3 must vote on it to be valid.
I also think that every new member should be educated about importance of participating in governance. I am thinking about it how it would be good. Any idea welcome ofcourse.  
I also think we could implement if 10 times in a row one does not vote he loses his right to be a DCDAO member, as participation is an important part.

**Comment by Cindy884** [comment]
Thks for your feedback. Actually DCDAO is implementing 2/3 voting requirement. You can learn more from this proposa:
https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0xf54f569ed893d5391b7b4dc716aa8b52bc99c2ac170c8beb43f98cb1b4ce7557

**Comment by SasoLithops** [comment]
Still needs to be implemented, as only 4 were enough to pass quorum in last voting and 13 of us now. I also recomend when divided by 3 and multiplied by 2 if the first round decimal number comes below 5 we keep the natural number in front and if it comes above 5 we add1 to number in front.

Example1 : 13 x 2/3= 8,7 so we need 9  to vote out of 13
Example2 : 17 x 2/3= 11,3 so we need 11  to vote out of 17

I will write most probably at the end of this month/start of next month if it is ok with you guys of course a tutorial how our governance works and questionare to all new DCDAO NFT holders so they learn and participate in our governance and seeing its value. 

As written above I recomend we take a snapshot of number of members just before vote goes live and 2/3 must vote on it to be valid.

**Reply by Cindy884** [reply]
hi saso, by now the quorum is 4, as for the growth of the DCDAO  NFT holder, we will adjust this number. This number means how many people have to participate in the voting.  We will update the number in the community if it changes.

**Reply by SasoLithops** [reply]
5 new members now😉
